src/components/CustomErrorComponent/CustomErrorComponent.tsx

```diff
@@ -6,13 +6,15 @@ export interface CustomErrorComponentProps {
 }
 
 export const CustomErrorComponent = ({ error }: CustomErrorComponentProps) => {
-  const isErrorResponse = (error: ErrorResponse | Error): error is ErrorResponse => error?.message === undefined;
+  const isErrorResponse = (error: ErrorResponse | Error): error is ErrorResponse => error instanceof ErrorResponse;
 
   return (
     <div className="flex flex-col w-full h-[80vh] justify-center items-center">
       <img src={fallingMan} alt="error image" />
       <p className="mt-4">Something went wrong</p>
-      <p>{isErrorResponse(error) ? JSON.parse(error.content).message : error.message}</p>
+      <p className={'first-letter:capitalize'}>
+        {isErrorResponse(error) ? JSON.parse(error.content).message : error.message}
```

Comment:
Just a thought as I'm not sure. But what happens here if `JSON.parse(error.content).message` throws due to parsing the content of the error response? Perhaps it could make sense to move it to useMemo or useEffect above and here just use the value set from said hooks? WDYT?